tests/components/totalconnect/snapshots/test_alarm_control_panel.ambr

```diff
@@ -44,7 +44,7 @@
       'location_id': 123456,
       'location_name': 'test',
       'low_battery': False,
-      'partition': 1,
+      'partition': '1',
```

Comment:
Let's first keep it an `int` by doing that in our code, and then have a PR where you make it a breaking change to make it a string

Reply:
done